plugins/trace_api_plugin/request_handler.cpp

```diff
@@ -26,105 +26,168 @@ namespace {
       return result;
 
    }
-
-   fc::variants process_actions(const std::vector<action_trace_v0>& actions, const data_handler_function& data_handler, const yield_function& yield ) {
+   fc::variants process_actions(trace_api_type_enum  e_action_trace, const vec_action_trace_variant  && actions, const data_handler_function_v0& data_handler_v0, const data_handler_function_v1 & data_handler_v1,  const return_handler_function & return_handler, const yield_function& yield ) {
```

Comment:
By making this use a template param for the trace version you can remove the need for this enum.

Reply:
Template was used.